### PR TITLE
using :?operator cts file is updated

### PR DESCRIPTION
**Pull request by Peddadevara-Divyasree**

### CODE REVIEW

File: flow/scripts/cts.tcl [hunk 1]
```diff
@@ -7,16 +7,9 @@ load_design 3_place.odb 3_place.sdc "Starting CTS"
...
-  set cluster_size "$::env(CTS_CLUSTER_SIZE)"
-} else {
-  set cluster_size 30
-}
-if {[info exist ::env(CTS_CLUSTER_DIAMETER)]} {
-  set cluster_diameter "$::env(CTS_CLUSTER_DIAMETER)"
-} else {
-  set cluster_diameter 100
-}
+#change ifelse to :?
```
Comment (maliberty) [inline flow/scripts/cts.tcl]: No need for this comment

File: input and output operations
Comment (maliberty) [inline input and output operations:1]: Do not rename

**Review by maliberty — CHANGES_REQUESTED** [review]
Thanks for contributing.  A few changes are required.  Also please see the DCO details link for information on fixing it.

**Comment by maliberty** [comment]
You are mixing all sorts of changes together.  Is this a PR that you expect to merge or just personal work?

**Comment by rovinski** [comment]
I too am confused about the purpose of this PR.

**Pull request closed without merging**